Tests/XCTestParametrizedMacroTests/SimpleValuesTests.swift

```diff
@@ -270,4 +270,31 @@ final class SimpleValuesTests: XCTestCase {
             macros: testMacros
         )
     }
+
+    func testParametrizeInputOutput_SingleObject() throws {
```

Comment:
It's rather single Int so we should change the test name for clarity.

Reply:
also removed some duplicated tests from other class.